pom.xml

```diff
@@ -429,6 +429,7 @@
         <version.jakarta.ejb.jakarta-ejb-api>4.0.1</version.jakarta.ejb.jakarta-ejb-api>
         <version.jakarta.enterprise.concurrent.jakarta-enterprise.concurrent-api>3.0.2</version.jakarta.enterprise.concurrent.jakarta-enterprise.concurrent-api>
         <version.jakarta.enterprise>4.0.1</version.jakarta.enterprise>
+        <version.jakarta.faces.jakarta-faces-api>4.0.0</version.jakarta.faces.jakarta-faces-api>
```

Comment:
@jasondlee I believe this should be 4.0.1. That's what it was before, and the diff between the two tells me the Mojarra code that produced them is different, with 4.0.1 being closer to the Mojarra was are using:

https://github.com/jakartaee/faces/compare/4.0.0-RELEASE...4.0.1-RELEASE#diff-f35103f068e191d5d8933637b2d4217d8f90a78728ccab1bada951a8d1c83590L111-R111